_includes/secondary-navigation.html

```diff
@@ -0,0 +1,135 @@
+{# ==========================================================================
+
+   secondary_navigation.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates markup for Header organism.
+
+   TODO: `active_nav_id` and `nav_items` parameters can be removed when
+         `_render_secondary_navigation_deprecated` is removed.
+
+   active_nav_id: Name of active navigation item.
+
+   nav_items:     The secondary navigation item structure.
+
+   ========================================================================== #}
+
+{% macro render(active_nav_id, nav_items) %}
+{% if page %}
```

Comment:
It's a wagtail variable. It'll always get ignored in this context, but I figure going forward it will be easier if this file matches as closely as possible what's in cfgov-refresh.